pages/common/source.md

```diff
@@ -0,0 +1,7 @@
+# source
+
+> Evaluate contents of file.
+
+- Causes fish to re-read its initialization file:
```

Comment:
This would only work if we are in `fish` shell. I think we should mention that.

Reply:
I've added bash examples alongside fish. Do you think it's enough?